donut/modules/flights/templates/flights.html

```diff
@@ -0,0 +1,19 @@
+{% extends "layout.html" %}
+{% block page %}
+
+<div class="container theme-showcase" role="main">
+	<div class="jumbotron">
+		<h1>Airport Carpools</h1>
+		{% if admin %}                        
+			<a class="btn btn-primary" href="{{ url_for('flights.settings') }}" role="button">Settings</a>
+		{% endif %}
+		{% if not visible %}
+			<h2>&#9992; Check back later &#9992;</h2>
```

Comment:
I think you should be able to put Unicode characters directly in the template rather than using HTML escapes